src/Error/GraphQLException.php

```diff
@@ -181,6 +207,26 @@ public function getOriginalErrorMessage(): ?string
         return null !== $this->originalException ? $this->originalException->getMessage() : null;
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function toArray(): array
+    {
+        return [
+            'message'   => $this->message,
+            'locations' => $this->getLocationsAsArray(),
+            'path'      => $this->path,
```

Comment:
Extensions should be included here when they're not null, right?

Reply:
Yes, I'll add it there.

Reply:
Done.

Reply:
Where did you add it?